Tests/Remora.Discord.Tests/Samples/Objects/MESSAGE_SNAPSHOT/MESSAGE_SNAPSHOT.optionals.json

```diff
@@ -0,0 +1,45 @@
+{
+  "message": {
```

Comment:
MessageSnapshot contains no optional fields - this file is redundant and can be deleted.